design/FY2016/OAControllerMakeover-Issue4663.md

```diff
@@ -0,0 +1,184 @@
+    ZoneAirDistribution,     !- Design Specification Zone Air Distribution Object Name 1
+    ZN_1_FLR_1_SEC_2,        !- Zone 2 Name
+    CM DSOA ZN_1_FLR_1_SEC_2,!- Design Specification Outdoor Air Object Name 2
+    ZoneAirDistribution,     !- Design Specification Zone Air Distribution Object Name 2
+    ZN_1_FLR_1_SEC_3,        !- Zone 3 Name
+    CM DSOA ZN_1_FLR_1_SEC_3,!- Design Specification Outdoor Air Object Name 3
+    ZoneAirDistribution,     !- Design Specification Zone Air Distribution Object Name 3
+    ZN_1_FLR_1_SEC_4,        !- Zone 4 Name
+    CM DSOA ZN_1_FLR_1_SEC_4,!- Design Specification Outdoor Air Object Name 4
+    ZoneAirDistribution,     !- Design Specification Zone Air Distribution Object Name 4
+    ZN_1_FLR_1_SEC_5,        !- Zone 5 Name
+    CM DSOA ZN_1_FLR_1_SEC_5,!- Design Specification Outdoor Air Object Name 5
+    ZoneAirDistribution;     !- Design Specification Zone Air Distribution Object Name 5
+
+ DesignSpecification:ZoneAirDistribution,
+    ZoneAirDistribution,     !- Name
+    1.2,                     !- Zone Air Distribution Effectiveness in Cooling Mode {dimensionless}
+    0.8,                     !- Zone Air Distribution Effectiveness in Heating Mode {dimensionless}
+    ,                        !- Zone Air Distribution Effectiveness Schedule Name
+    0.0;                     !- Zone Secondary Recirculation Fraction {dimensionless}
+
+ DesignSpecification:OutdoorAir,
+    CM DSOA ZN_1_FLR_1_SEC_1,!- Name
+    Sum,                     !- Outdoor Air Method
+    0.0100,                  !- Outdoor Air Flow per Person {m3/s-person}
+    0.0000,                  !- Outdoor Air Flow per Zone Floor Area {m3/s-m2}
+    ,                        !- Outdoor Air Flow per Zone {m3/s}
+    ,                        !- Outdoor Air Flow Air Changes per Hour {1/hr}
+    MinOA_MotorizedDamper_Sched;  !- Outdoor Air Flow Rate Fraction Schedule Name
+
+```
+
+## Proposed Changes ##
+
+1. Move these three fields from `Controller:MechanicalVentilation` to `Controller:OutdoorAir`:
+```
+    ALWAYS_ON,               !- Availability Schedule Name
+    Yes,                     !- Demand Controlled Ventilation
+    VentilationRateProcedure,!- System Outdoor Air Method
+```
+
+2. Rename `Controller:MechanicalVentilation` to `Controller:OutdoorAir:ZoneList` and make it optional.  If all of the zones have a Sizing:Zone object, then the designspec info will be taken from there.
+
+3. Change the autosizing for `Controller:OutdoorAir` Minimum Outdoor Air Flow Rate to be zero if one of the advanced methods (VRP, DCV, IAQ) of control is selected.  *Or maybe autosize to the non-per-person flow?*
+
+4. Make the Maximum Fraction of Outdoor Air Schedule Name be king - OA fraction can never be greater than the current schedule value.
+
+5. Change the advanced methods (VRP, DCV, IAQ) to use the `DesignSpecification:OutdoorAir` Outdoor Air Flow Rate Fraction Schedule.  Currently it is ignored.
+
```

Comment:
It's a multiplier on the base OA flow rate, so if it's zero then OA should go to zero.  ZoneSum will have no problem with this, but I will need to test the other methods to make sure the logic works for them.  It does raise a question for IAQ method.